packages/mdc-textfield/mdc-text-field.scss

```diff
@@ -157,48 +155,15 @@
     }
   }
 
-  &.mdc-text-field--focused .mdc-text-field__idle-outline,
-  .mdc-text-field__label--float-above ~ .mdc-text-field__idle-outline {
-    opacity: 0;
-  }
-
-  &.mdc-text-field--focused .mdc-text-field__outline,
-  .mdc-text-field__label--float-above ~ .mdc-text-field__outline {
-    opacity: 1;
-  }
-
-  &.mdc-text-field--focused .mdc-text-field__outline-path {
-    @include mdc-theme-prop(stroke, primary);
-
-    stroke-width: 2px;
-  }
-
-  &.mdc-text-field--disabled {
-    @include mdc-theme-prop(color, $mdc-text-field-light-placeholder);
-
-    .mdc-text-field__input {
-      border-bottom: none;
-    }
-
-    .mdc-text-field__outline-path {
-      stroke: $mdc-text-field-outlined-disabled-border;
-      stroke-width: 1px;
-    }
-
-    .mdc-text-field__idle-outline {
-      border-color: $mdc-text-field-outlined-disabled-border;
-    }
-
-    .mdc-text-field__label {
-      bottom: 20px;
-    }
+  // stylelint-enable plugin/selector-bem-pattern
+}
 
-    .mdc-text-field__icon {
-      @include mdc-theme-prop(color, $mdc-text-field-disabled-icon-on-light);
-    }
-  }
+.mdc-text-field--outlined.mdc-text-field--focused {
+  @include mdc-text-field-focused-outline_();
+}
 
-  // stylelint-enable plugin/selector-bem-pattern
+.mdc-text-field--outlined.mdc-text-field--focused {
```

Comment:
I think you meant for this block to be `--disabled` and the mixin to be `mdc-text-field-outlined-disabled_` (see comment on mixin above)

Reply:
Yes, correct! That was confusing naming on my part. Thanks for catching!